links/CompressorLink/CompressorLink.sc

```diff
@@ -38,7 +38,8 @@ CompressorLink : Link {
 
     ar { | input, id = "" |
         var signal = input;
-        signal = Compander.ar(signal, signal,
+		var monomix = Mix(input); // Mix signal to mono, preventing stereo imaging distortion (same compression for both channels)
```

Comment:
Looks like some weird indenting here.

Also should we use `Mix.kr()` for control rate?